hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/AbstractHoodieWriteClient.java

```diff
@@ -712,24 +742,43 @@ public void rollbackInflightCompaction(HoodieInstant inflightInstant, HoodieTabl
   }
 
   /**
-   * Cleanup all pending commits.
+   * Rollback all failed commits.
    */
-  private void rollbackPendingCommits() {
+  public void rollbackFailedCommits() {
```

Comment:
Intention is to make `rollback()` private, since this is the first release, I've only marked it as @deprecated. Going forward people should use the following 2 API's : `restore()` to revert successful commits and `rollbackFailedCommits()` to delete failed writes.